go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/deckarep/golang-set/v2 v2.3.0
 	github.com/google/go-cmp v0.5.9
 	github.com/vmware-tanzu/velero v1.12.0
+	k8s.io/klog v1.0.0
```

Comment:
we already have `k8s.io/klog/v2 v2.90.0 // indirect` as indirect dependency. Can not we use that instead of this one?